product/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length = 50)
+
+    class Meta:
+        db_table = 'categories'
+
+class Collection(models.Model):
+    name  = models.CharField(max_length = 50)
+    image = models.URLField(max_length = 500)
+    video = models.URLField(max_length = 500)
+
+    class Meta:
+        db_table = 'collections'
+
+class Product(models.Model):
+    category          = models.ForeignKey(Category, on_delete = models.CASCADE)
+    name              = models.CharField(max_length = 50)
+    description_image = models.URLField(max_length = 5000)
+    price             = models.DecimalField(max_digits=6, decimal_places=4)
+    collection        = models.ForeignKey(Collection, on_delete = models.CASCADE)
+    stock             = models.CharField(max_length = 50)
+    tag               = models.ManyToManyField('Tag', through = 'ProductTag')
+
+    class Meta:
+        db_table = 'products'
+        
+class ProductTag(models.Model):
+    product  = models.ForeignKey(Product, on_delete = models.CASCADE)
+    tag = models.ForeignKey('Tag', on_delete = models.CASCADE)
+    
+    class Meta:
+        db_table = 'products_tags'
+
+class Color(models.Model):
+    name    = models.CharField(max_length = 50)
+    product = models.ManyToManyField(Product)
```

Comment:
table이 ManyToMany 관계의 경우, 위에서 사용하신 방법은 Django에서 내부적으로 table들의 중간 테이블을 만들어줍니다.

이 방법을 사용하지 않고, 실제로 중간 테이블을 만들어서 ManyToMany 관계를 만들어주세요! 아래 링크를 참고해주세요.
https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.ManyToManyField


<img width="912" alt="스크린샷 2020-08-21 오전 10 40 09" src="https://user-images.githubusercontent.com/53214510/90842604-b0dd8e00-e39a-11ea-8b17-fce1a33798c5.png">

```suggestion
class Color(models.Model):
```
```suggestion
    product = models.ManyToManyField(Product)
```